minard/templates/burst.html

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.html" %}
+{% block title %}Burst Trigger{% endblock %}
+{% block head %}
+{{ super() }}
+<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/pca.css') }}" media="screen">
+{% endblock %}
+{% block body %}
+{{ super() }}
+
+    <div class="container">
+        <h2>Supernova Burst Trigger </h2>
```

Comment:
Everything should be wrapped in a row and column. Here, since you're not using the grid layout you can just add a:

```
<div class="row">
    <div class="col-md-12">
        <h2>Supernova Burst Trigger</h2>
        <!-- everything else goes here -->
    </div>
</div>
```